cypress/e2e/chi-vue/transfer-list.cy.js

```diff
@@ -0,0 +1,184 @@
+const BUTTON_CLASS = 'chi-button';
+const BACKDROP_CLASS = 'chi-backdrop';
+const CLOSED_CLASS = '-closed';
+const MODAL_CLASS = 'chi-modal';
+const MODAL_TITLE_CLASS = 'chi-modal__title';
+const TOOLTIP_CLASS = 'chi-tooltip';
+ 
+const TRANSFER_LIST_CLASSES = {
+  TRANSFER_LIST_ELEMENT: 'chi-transfer-list',
+  TRANSFER_LIST_COLUMN: 'chi-transfer-list__column',
+  TRANSFER_LIST_TITLE: 'chi-transfer-list__title',
+  TRANSFER_LIST_SEARCH: 'chi-transfer-list__search',
+  TRANSFER_LIST_MENU: 'chi-transfer-list__menu',
+  TRANSFER_LIST_FOOTER: 'chi-transfer-list__footer'
+};
+
+describe('Transfer List', () => {
+  before(() => {
+    cy.visit('tests/chi-vue/transfer-list.html');
+  });
+
+  describe('Base', () => {
+    beforeEach(() => {
+      cy.get("[data-cy='transfer-list-base']").as('transfer-list');
+    });
+
+    it(`should have class ${TRANSFER_LIST_CLASSES.TRANSFER_LIST_ELEMENT}`, () => {
```

Comment:
Uppercase, check in every test

```suggestion
    it(`Should have class ${TRANSFER_LIST_CLASSES.TRANSFER_LIST_ELEMENT}`, () => {
```